### PR TITLE
Template report

### PR DESCRIPTION
**Pull request by balthazarneveu**
Please review your information @ManalAkh  & @InesVATI  . Then we squash to have a clean blank document.

### CODE REVIEW

File: report/report.tex [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+% \acmDOI{XXXXXXX.XXXXXXX}
+
+%% These commands are for a PROCEEDINGS abstract or paper.
+\acmConference[MVA 2023]{MVA 2023}{December
+  2023}{Paris, FR}
+
+\acmPrice{15.00}
+\acmISBN{978-1-4503-XXXX-X/18/06}
+
+
```
Comment (balthazarneveu) [inline report/report.tex:23]: We may want to get rid of all these long comments

Comment (balthazarneveu) [reply report/report.tex:23]: Done

File: report/report.tex [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+%%
+%% AUTHORS
+\author{Manal Akhannous}
+\affiliation{%
+  \institution{ENPC}
+  \streetaddress{Cité Descartes, 8 Av Blaise Pascal}
+  \city{Champs-sur-Marne}
+  \country{France}
+}
+\email{manal.akhannous@enpc.fr}
```
Comment (InesVATI) [inline report/report.tex:39]:
```suggestion
\email{manal.akhannous@eleves.enpc.fr}
```

File: report/report.tex [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+\author{Ines Vati}
+\affiliation{%
+  \institution{ENPC}
+  \streetaddress{Cité Descartes, 8 Av Blaise Pascal 6 et}
+  \city{Champs-sur-Marne}
+  \country{France}
+}
+\email{ines.vati@enpc.fr}
```
Comment (InesVATI) [inline report/report.tex:48]:
```suggestion
\email{ines.vati@eleves.enpc.fr}
```

**Review by InesVATI — APPROVED** [review]

**Review by ManalAkh — APPROVED** [review]

**Review by InesVATI — APPROVED** [review]

**Pull request merged**